setup.py

```diff
@@ -15,7 +15,7 @@
     'astropy>=3.2.1',
     'astroquery>=0.3.9',
     'bandit',
-    'bokeh',
+    'bokeh==2.4.3',
```

Comment:
Set them equal to what they are in the environment files?

Reply:
So bokeh is currently at v3.0+ but of course some of the modules/methods have moved around. Particularly the `Panel` which caused some issues for `Exoctk`. I figured you were pinning it to avoid new versions from installing.